webapp-django/crashstats/crashstats/static/crashstats/js/socorro/admin.js

```diff
@@ -1,3 +1,6 @@
+/* eslint-disable no-unused-vars */
```

Comment:
Instead of disabling this rule, we can use the [exported](https://eslint.org/docs/rules/no-unused-vars#exported) comment to explicitly mark the functions as being exported. That lets us continue to look for unused variables while ignoring stuff we mean to export.

Sames goes for the other spots that you're disabling `no-unused-vars`.